apps/api/lib/api/client/views/relay.ex

```diff
@@ -0,0 +1,32 @@
+defmodule API.Client.Views.Relay do
+  alias Domain.Relays
+
+  def render_many(relays, expires_at) do
+    Enum.flat_map(relays, &render(&1, expires_at))
+  end
+
+  def render(%Relays.Relay{} = relay, expires_at) do
+    [
+      maybe_render(relay, expires_at, relay.ipv4),
+      maybe_render(relay, expires_at, relay.ipv6)
+    ]
+    |> Enum.reject(&is_nil/1)
+  end
+
+  defp maybe_render(%Relays.Relay{}, _expires_at, nil), do: nil
+
+  defp maybe_render(%Relays.Relay{} = relay, expires_at, address) do
+    %{
+      username: username,
+      password: password,
+      expires_at: expires_at
+    } = Relays.generate_username_and_password(relay, expires_at)
+
+    %{
+      uri: "stun:#{address}:#{relay.port}",
```

Comment:
@thomaseizinger @jamilbk This is the implementation of our relay auth spec, should be up to date with the latest changes.

Reply:
```suggestion
      uri: "turn:#{address}:#{relay.port}",
```

Nit: stun will not require authentication, only the turn messages need that although it might be good to send a `stun:` URI back as well. Then, if you ever want to host them somewhere else, clients don't need to change.